LeoBirds/Engine/Vehicle.ixx

```diff
@@ -0,0 +1,63 @@
+﻿export module Engine.Vehicle;
+
+import <memory>;
+import <stdexcept>;
+import <SFML/System/Vector2.hpp>;
+
+import FLib.DrawableImage;
+import BoatCharacteristics;
+import GunCharacteristics;
+
+namespace birds
+{
+    export class Vehicle
+    {
+    public:
+        //Constructor of gun
```

Comment:
use documentation template
Use something line : "Creates a new instance of a vehicule"